app/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The structure looks similar to aadharr_cardlist some layout part should refactored and put in common